README.md

```diff
@@ -108,7 +108,9 @@ To build Content-Security-Policy policies, we recommend using the [Laboratory CS
 
 #### Report-Only Policies
 
-To send a [Content-Security-Policy-Report-Only header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only), use the exact same process described above for the ordinary CSP policies with the alternative filter `altis.security.browser.report_only_content_security_policies`. An external service must be used to ingest the reports from the Report-Only policies. The external service will provide you with a reporting URL which can then be defined by adding a `report-uri` directive with the appropriate URL for processing reports.
+To send a [Content-Security-Policy-Report-Only header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only), use the exact same process described above for the ordinary CSP policies with the alternative filter `altis.security.browser.report_only_content_security_policies`.
+
+An external service must be used to ingest the reports from Report-Only policies. The external service will provide you with a reporting URL which you can use by adding a `report-uri` directive with the appropriate URL for processing reports.
```

Comment:
I think an example of the code would help here too, perhaps with a real service like uriports.com

Reply:
👍 makes sense. I was looking at report-uri.com as it's run by Troy Hunt and folks and has a free tier, but hadn't heard of uriports. Looks like it has some better pricing after the low tier 🤔

Reply:
```suggestion
An external service must be used to ingest the reports from Report-Only policies. The external service will provide you with a reporting URL which you can use by adding a `report-uri` directive with the appropriate URL for processing reports.

As an example, you can add a reporting directive to your Report-Only policies by filtering the policies array:
```php
add_filter( 'altis.security.browser.report_only_content_security_policies', function ( array $policies ) : array {
	$policies['report-uri'] = 'https://example.uriports.com/reports';
	return $policies;
} );
```
```